aws_endpoint: "a6k20qx3hahii-ats.iot.eu-west-3.amazonaws.com"
aws_port: 8883

ca_name: "AmazonRootCA1.pem"
cert_suffix: "-certificate.pem.crt"
key_suffix: "-private.pem.key"

private_bucket: 's3-niryo-private'
public_bucket: 's3-niryo-public'

s3_programs_dir: 'programs'